nixos/amis.nix

```diff
@@ -0,0 +1,295 @@
+let self = {
```

Comment:
couldn't we also get this file from `<nixpkgs/nixos/modules/virtualisation/ec2-amis.nix>` or `${pkgs.path}/nixos/modules/virtualisation/ec2-amis.nix`?